src/yunohost/user.py

```diff
@@ -453,3 +441,10 @@ def _convertSize(num, suffix=''):
             return "%3.1f%s%s" % (num, unit, suffix)
         num /= 1024.0
     return "%.1f%s%s" % (num, 'Yi', suffix)
+
+
+def _hash_user_password(password):
+    char_set = string.ascii_uppercase + string.ascii_lowercase + string.digits + "./"
+    salt = ''.join([random.SystemRandom().choice(char_set) for x in range(16)])
```

Comment:
Maybe we can add a comment here that this is a crypto-secure(?) source of entropy (if you actually checked it is ;))